libs/platform/src/lib/components/form/form-group/form-field/form-field.component.ts

```diff
@@ -294,9 +283,13 @@ export class FormFieldComponent implements FormField, AfterContentInit, AfterVie
      * Add FormField to FormGroup
      */
     private addToFormGroup(): void {
-        if (!this.formGroupContainer) {
+        // prevent adding wrapped fields into field group into form group container
+        const isFieldGroup = this._elementRef.nativeElement.parentElement.tagName.toLowerCase() === 'fdp-form-field-group';
```

Comment:
Let's use @Host @optional injection approach to understand if we are into FormFieldGroup